composer.json

```diff
@@ -18,17 +18,16 @@
         }
     ],
     "require": {
-        "php":                  "^7.3.0",
-        "doctrine/orm":         "^2.7.4",
+        "php":                  "^7.4.0|| ^8.0",
+        "doctrine/orm":         "^2.7.4|| ^2.8",
```

Comment:
Can we reduce this to `^2.8`?

Reply:
Sure I'll do this tomorrow 👍

Reply:
Also, iterate method on AbstractQuery is deprecated on 2.8 and report to toIterable.
This refer to #89 which propose to replace Traversable by iterable, can we after this PR merged do this? This will lead to a BC and 3.0 will must be tagged.

Reply:
Done